GVRf/Framework/gradle.properties

```diff
@@ -15,6 +15,7 @@
 #Thu May 05 12:55:40 PDT 2016
 android.useDeprecatedNdk=true
 org.gradle.jvmargs=-Xmx2048M
+GVRF_BACKEND=backend_oculus
```

Comment:
I am not sure why the Extension projects need that. If they use an .aar it shouldn't matter. Or does it?